style.css

```diff
@@ -3772,6 +3816,54 @@ ul.footer-social li {
 		margin: 1.2rem 0 0 2.4rem;
 	}
 
+	/* SITE PAGINATION */
```

Comment:
There's a bigger comment heading called Site Pagination a couple of lines up in the CSS file which you can move this block into, for consistency.